### PR TITLE
add gvcf file for cttsov2 and add content disposition header for s3 p…

### PR DESCRIPTION
**Pull request by raylrui**
Fix:  
1. include `.gvcf` files in the analysis tables
2. add 'Content-Disposition' header in 'preview' s3 presignurl api, and change default presignurl behaviour to download as attachment, this will couple PR in `data-portal-apis` and `libumccr.aws.libs3.presign_s3_file` to be able add 'content_disposition' params
3. fix some other type issues

Apis PR: https://github.com/umccr/data-portal-apis/pull/705
Libumccr PR: https://github.com/umccr/libumccr/pull/28

Reference: 
- about "Content-Disposition" header: https://httpwg.org/specs/rfc6266.html#header.field.definition
- about boto3 presign url pass disposition header: https://github.com/boto/boto3/issues/356#issuecomment-155919331
https://github.com/boto/boto3/issues/610#issuecomment-443878760

### CODE REVIEW

File: src/containers/subjects/AnalysisResultTable/index.tsx [hunk 1]
```diff
@@ -519,7 +524,9 @@ function groupResultsData({
     (r) => r.key.includes('cttsov2') && r.key.endsWith('bam')
   );
   const cttsov2Vcfs = results_cttsov2.filter(
-    (r) => r.key.includes('cttsov2') && (r.key.endsWith('.vcf') || r.key.endsWith('.vcf.gz'))
+    (r) =>
+      r.key.includes('cttsov2') &&
+      (r.key.endsWith('.vcf') || r.key.endsWith('.gvcf') || r.key.endsWith('.vcf.gz'))
```
Comment (victorskl) [inline src/containers/subjects/AnalysisResultTable/index.tsx:529]: Can you also add `.gvcf.gz` here, pls.

**Review by victorskl — APPROVED** [review]
LGTM. Just one minor comment to attend.

File: src/containers/subjects/AnalysisResultTable/index.tsx (hunk 1 above)
Comment (raylrui) [reply src/containers/subjects/AnalysisResultTable/index.tsx:529]: ah, yes

**Comment by raylrui** [comment]
Merging to see

**Pull request merged**